File: Retrofit/app/src/main/java/com/example/arnassmicius/retrofit/ReposAdapterViewHolder.java
package com.example.arnassmicius.retrofit;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

/**
 * Created by arnas on 18.2.16.
 */

public class ReposAdapterViewHolder extends RecyclerView.ViewHolder {

    final TextView listItem;

    public ReposAdapterViewHolder(View itemView) {
        super(itemView);

        listItem = itemView.findViewById(R.id.list_item);
    }
}

File: Retrofit/app/src/main/java/com/example/arnassmicius/retrofit/ReposAdapterViewHolder.kt
package com.example.arnassmicius.retrofit

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

/**
 * Created by arnas on 18.2.16.
 */

class ReposAdapterViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    internal val listItem: TextView = itemView.findViewById(R.id.list_item)

}
